packages/gasket-utils/docs/api.md

```diff
@@ -0,0 +1,207 @@
+
+## Classes
+
+Name | Description
+------ | -----------
+[PackageManager] | Wrapper class for executing commands for a given package manager
+
+## Functions
+
+Name | Description
+------ | -----------
+[applyEnvironmentOverrides(gasketConfig, config, \[localFile\])] | Normalize the config by applying any environment or local overrides
+[runShellCommand(cmd, argv, options, \[debug\])] | Promise friendly wrapper to running a shell command (eg: git, npm, ls)which passes back any { stdout, stderr } to the error thrown.
+[tryRequire(path)] | Tries to require a module, but ignores if it is not found.If not found, result will be null.
+
+
+## PackageManager
+
+Wrapper class for executing commands for a given package manager
+
+**Kind**: global class  
+
+* [PackageManager]
+    * [new PackageManager(options)]
+    * _instance_
+        * [.exec(cmd, args)]
+        * [.link(packages)]
+        * [.install(args)]
+        * [.info(args)]
+    * _static_
+        * [.spawnNpm(argv, spawnWith)]
+        * [.spawnYarn(argv, spawnWith)]
+
+
+### new PackageManager(options)
+
+
+| Param | Type | Description |
+| --- | --- | --- |
+| options | `object` | Options |
+| \[options.packageManager\] | `string` | Name of manager, either `npm` (default) or `yarn` |
+| options.dest | `string` | Target directory where `node_module` should exist |
+| \[options.npmconfig\] | `string` | DEPRECATED Path to userconfig |
+
+
+### packageManager.exec(cmd, args)
+
+Executes npm in the application directory `this.dest`.
+This installation can be run multiple times.
```

Comment:
This is fine for now, but given it would improve the `util`-ity of this package if this also included example usage.